app/assets/stylesheets/application.scss

```diff
@@ -41,8 +46,67 @@ $max-width: 1000px;
 }
 
 a {
-  text-decoration: underline;
+  text-decoration: none;
   &:hover {
-    text-decoration: none;
+    text-decoration: underline;
   }
 }
+
+.sorting-icon {
+  width: 10px;
+  margin-left: 5px;
+  vertical-align: middle;
+}
+
+.tag {
+  margin-top: 10px;
+}
+.tag.active {
+  background-color:#dae7b8;
+  color:#526D0D;
+  border:1px solid #B3CC6E;
+}
+
+.tag.inactive {
+  background-color:#FFDEDE;
+  color:#BB0B0B;
+  border:1px solid #FF0000;
+}
+
+.add_record_button {
+  float: right;
+}
+
+.filters-body
+{
+  justify-content: space-evenly;
+}
+
+.filters__select {
+  max-width:none; 
+}
+
+.employee-info-table {
+  background-color: #ffffff;
+}
+
+.employee-info-table tr.table-column {
+  padding-bottom: 15px;
+  background-color: #ffffff;
+
+}
+
+.employee-info-table tr.column-value {
+  padding-top: 0px;
+  background-color: #ffffff;
+}
+
+.employee-detail {
+  border-bottom: 1px solid #999999;
+  padding-bottom: 40px;
+  color: #999999;
+}
+
+.qualifications tr.table-column {
+  border-bottom: 1px solid #999999;
+}
```

Comment:
if these are extensions to existing design library files coud you add them to the relevant files.

If they're new we should try to genericise them as much as possible as we build out more of the pages